src/components/client/EditClient.js

```diff
@@ -66,6 +66,7 @@ class EditClient extends Component {
         <Form>
           <Row>
             <h1>{t('client.contact')}</h1>
+            <h4>{t('createdOn')} {new Date(client.createdOn).toLocaleDateString()}</h4>
```

Comment:
Dit gaat de datum tonen in de locale van de browser. Bij mij is dat bijvoorbeeld vandaag "3/25/2019".
Dit is verwarrend - omdat de rest van de applicatie in NL is.
MomentJS is reeds een dependency, je kan deze gebruiken om de datum te formatten in iets dat logischer is in België.